lib/liquid/parse_context.rb

```diff
@@ -19,7 +21,6 @@ def partial=(value)
       @partial = value
       @options = value ? partial_options : @template_options
       @error_mode = @options[:error_mode] || Template.error_mode
-      value
```

Comment:
Why was this removed? That looks like it was here on purpose

Reply:
This one is a complaint that a return value for a void function. I'm not familiar with Ruby but i'm assuming because this is an assignment method it can't return a value